furusiya/model/entities/enemies/salamander.py

```diff
@@ -0,0 +1,5 @@
+from model.entities.game_object import GameObject
+
+
+class Salamander(GameObject):
```

Comment:
Do we really need this class?

Reply:
Now? Nope. It is used to implement fire-shooting stuff in the other PR.